src/scripts/star-rating.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+var star1 = document.getElementsByClassName('star-1');
+var star2 = document.getElementsByClassName('star-2');
+var star3 = document.getElementsByClassName('star-3');
+var star4 = document.getElementsByClassName('star-4');
+var star5 = document.getElementsByClassName('star-5');
+
+// Function that marks first star //
+
+const firstStar = function() {
+  for(let i = 0; star1.length > i; i++){
+    star1[i].addEventListener('click', function() {
```

Comment:
This solution is a little bit hard to read and may be hard to maintain in the future. When you get more familiar with JS you may give it a while and try to figure out a more generic solution.

In this example - https://codepen.io/pxsx/pen/Kbjapv - hover effect has been made with pure CSS and clicking has been handled with two loops.